src/main/java/org/deepsymmetry/beatlink/VirtualRekordbox.java

```diff
@@ -474,11 +502,8 @@ private DeviceUpdate buildUpdate(DatagramPacket packet) {
                         }
                     }
                 } else if (data[0x25] == METADATA_TYPE_IDENTIFIER_SONG_CHANGE) {
-                    try {
-                        requestPSSI();
-                    } catch (IOException e) {
-                        logger.warn("Cannot send PSSI request");
-                    }
+                    // This was one way to detect a track change, but the packet is sent too slow after a track change,
+                    // instead we detect for an ID change in between status packets in the CDJ_STATUS case above.
```

Comment:
Should we perhaps just delete this (now empty) `else if` entirely? If we need it back for some reason someday, it will be in git history.

Reply:
Yep, I left it here as sort of "hey this packet is available here", but I could probably just add a note to my [repo](https://github.com/kyleawayan/opus-quad-pro-dj-link-analysis)

Reply:
Probably the most important thing is to make sure that information has been captured in the [protocol analysis](https://djl-analysis.deepsymmetry.org/).

Reply:
Ah that’s right, I forgot you had your own notes about this, that’s fine! 😄